DSL/Query/ConstantScoreQuery.php

```diff
@@ -29,16 +32,17 @@ class ConstantScoreQuery implements BuilderInterface
     /**
      * @var BuilderInterface
      */
-    private $filterOrQuery;
+    private $query;
 
     /**
-     * @param BuilderInterface $filterOrQuery
+     * @param BuilderInterface $query
      * @param array            $parameters
+     * @param string           $dslType
      */
-    public function __construct(BuilderInterface $filterOrQuery, array $parameters = [])
+    public function __construct(BuilderInterface $query, array $parameters = [], $dslType = self::USE_FILTER)
```

Comment:
Maybe default value should be ``self::USE_QUERY`` since it is ``ConstantScoreQuery`` query.